tests/test_merge.py

```diff
@@ -64,6 +64,26 @@ def test_continue_run_multiple_conflicts_per_patch(self):
             self.assertEqual(patch(m.unified_patches, lca),
                              expected_value)
 
+    def test_continue_run_multiple_conflicts_per_patch_two(self):
```

Comment:
This name needs to be changed. The 'two' is funny.

Reply:
I didn't even noticed 👍